src/main/msp/msp.c

```diff
@@ -1833,32 +1819,18 @@ case MSP_NAME:
 
         // API 1.41 - Add multi-gyro indicator, selected gyro, and support for separate gyro 1 & 2 alignment
         sbufWriteU8(dst, getGyroDetectionFlags());
-#ifdef USE_MULTI_GYRO
-        sbufWriteU8(dst, gyroConfig()->gyro_to_use);
+        sbufWriteU8(dst, 0); // deprecated gyro_to_use
+#if GYRO_COUNT > 1
         sbufWriteU8(dst, gyroDeviceConfig(0)->alignment);
         sbufWriteU8(dst, gyroDeviceConfig(1)->alignment);
 #else
-        sbufWriteU8(dst, GYRO_CONFIG_USE_GYRO_1);
         sbufWriteU8(dst, gyroDeviceConfig(0)->alignment);
         sbufWriteU8(dst, ALIGN_DEFAULT);
 #endif
         // Added in MSP API 1.47
-        switch (gyroConfig()->gyro_to_use) {
-#ifdef USE_MULTI_GYRO
-        case GYRO_CONFIG_USE_GYRO_2:
-            sbufWriteU16(dst, gyroDeviceConfig(1)->customAlignment.roll);
-            sbufWriteU16(dst, gyroDeviceConfig(1)->customAlignment.pitch);
-            sbufWriteU16(dst, gyroDeviceConfig(1)->customAlignment.yaw);
-            break;
-#endif
-        case GYRO_CONFIG_USE_GYRO_BOTH:
-            // for dual-gyro in "BOTH" mode we only read/write gyro 0
-        default:
-            sbufWriteU16(dst, gyroDeviceConfig(0)->customAlignment.roll);
-            sbufWriteU16(dst, gyroDeviceConfig(0)->customAlignment.pitch);
-            sbufWriteU16(dst, gyroDeviceConfig(0)->customAlignment.yaw);
-            break;
-        }
+        sbufWriteU16(dst, gyroDeviceConfig(0)->customAlignment.roll);
+        sbufWriteU16(dst, gyroDeviceConfig(0)->customAlignment.pitch);
+        sbufWriteU16(dst, gyroDeviceConfig(0)->customAlignment.yaw);
```

Comment:
We loose the ability to set customAlignment for second gyro.
Think we want to set these for all available gyros independent.

Reply:
Yes, we will want to set it independent, however, we probably just need to update MSP so that we can choose which IMU we want to modify and then modify it on its own, but this will require an update to MSP.

Reply:
Do not think we need a helper function (in msp) as we need save AND reboot to apply settings.

Reply:
So should I make it do firstEnabledGyro()?